echopype/convert/utils/ek_raw_parsers.py

```diff
@@ -522,7 +522,17 @@ def __init__(self):
                 ("roll", "f"),
                 ("pitch", "f"),
                 ("heading", "f"),
-            ]
+            ],
+            1: [
```

Comment:
Could you add in the docstring regarding MRU1 type in addition to MRU0?

Reply:
Updated!